srcpkgs/oil/template

```diff
@@ -27,8 +27,9 @@ pre_build() {
 do_install() {
 	# Install binaries
 	vbin _bin/oil.ovm
-	# Symlink osh, oshc to oil.ovm.
+	vman doc/osh.1
+	# Symlink osh, oil to oil.ovm.
 	cd "${DESTDIR}"/usr/bin
 	ln -s oil.ovm osh
-	ln -s oil.ovm oshc
+	ln -s oil.ovm oil
```

Comment:
Not sure it makes sense to break people's stuff like this... Why not keep `oshc` around?

Reply:
From the project author:

> `oshc translate` and `oshc` are mostly experimental.  The project has moved away from source translation in favor of simply running bash scripts with OSH, and allowing OSH and Oil to seamlessly interoperate.
> 
> I think the only reason to keep them would be to expose the experiments to users and hope someone takes over :)
> 
> But it should be fine to delete them; I don't think anyone is using them.  They're not really documented

https://oilshell.zulipchat.com/#narrow/stream/264891-oil-help/topic/Removing.20.60oshc.60.20from.20Void's.20packages/near/252874976